pkg/resources/sources/options.go

```diff
@@ -3,7 +3,55 @@
 
 package sources
 
+import (
+	"fmt"
+	"strconv"
+)
+
 type Options struct {
-	Sources        []string
-	OvercommitVCPU float64
+	Sources            []string
+	OvercommitVCPU     float64
+	ReservedMemorySize MemorySize
+	BlockedHugepages   uint64
+}
+
+// MemorySize is a custom type to handle memory sizes in human-readable format.
+type MemorySize uint64
```

Comment:
why? cannot you use any existing library e.g.: https://github.com/docker/go-units/blob/master/size.go

Reply:
Used